pkg/workflow/spec/convert/convert.go

```diff
@@ -0,0 +1,45 @@
+package convert
```

Comment:
I kind of wonder if this should be in an `internal` package - i.e., are we treating this as an implementation detail of the `convertMarkdown` function or do we want to actually expose it as something we expect people to be able to use directly?

Reply:
Hard to say.  Right now it's technically generally useful, but exposing would somewhat indicate it belongs in another library.

Reply:
As we expand our function library as mentioned above, I would suggest we'll probably have trend towards having functions as a wrapper around library code with well-defined interfaces and exposed implementations.  But we'll see.